src/main/java/com/odysseusinc/arachne/portal/config/interceptors/RequestInterceptor.java

```diff
@@ -57,7 +57,7 @@ public void afterCompletion(HttpServletRequest request, HttpServletResponse resp
         Long queryCount = interceptor.getCount();
         interceptor.clearCounter();
         time.remove();
-        LOGGER.info("[Time: {} ms] [Queries: {}] {} {}", duration, queryCount, request.getMethod(), request.getRequestURI());
+        LOGGER.debug("[Time: {} ms] [Queries: {}] {} {}", duration, queryCount, request.getMethod(), request.getRequestURI());
```

Comment:
You can off these message in settings: portal.useQueryInterceptor

Reply:
In any case this information related to debug and not needed for info log-level